### PR TITLE
list of goods

### PR DESCRIPTION
**Pull request by donizer**
[DEMO LINK](https://donizer.github.io/react_list-of-goods-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,45 +16,129 @@ export const goodsFromServer = [
...
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
```
Comment (loralevitska) [inline src/App.jsx:64]: Rename boolean variable according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig). Fix all cases if present
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```

File: src/App.jsx [hunk 2]
```diff
@@ -14,45 +16,129 @@ export const goodsFromServer = [
...
+                {
+                  'is-light': true,
+                },
+              )}
+              onClick={reset}
+            >
+              Reset
+            </button>
+          )
+          : ''}
```
Comment (loralevitska) [inline src/App.jsx:133]:
```suggestion
        {(sortField || reversed) && (
            <button
              type="button"
              className={classNames(
                'button is-danger',
                {
                  'is-light': true,
                },
              )}
              onClick={reset}
            >
              Reset
            </button>
          )}
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost done) Let's improve

File: src/App.jsx [hunk 3]
```diff
@@ -14,45 +16,128 @@ export const goodsFromServer = [
...
+            {
+              'is-light': !isReversed,
+            },
+          )}
+          onClick={() => setIsReversed(current => !current)}
+        >
+          Reverse
+        </button>
+
+        {(sortField !== '' || isReversed !== false)
```
Comment (TarasHoliuk) [inline src/App.jsx:118]: the main idea of the related components was the following: you shouldn't compare the boolean variable with `false` and the string variable with an empty string: [conditional rendering](https://react.dev/learn/conditional-rendering#logical-and-operator-)
Empty string is a [falsy](https://www.freecodecamp.org/news/falsy-values-in-javascript/) value

Comment (TarasHoliuk) [inline src/App.jsx:118]:
```suggestion
        {(sortField || isReversed)
```

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]

**Review by loralevitska — APPROVED** [review]
Good job!

**Pull request closed without merging**